src/main/java/com/khutircraftubackend/jwt/exception/JwtExceptionHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.khutircraftubackend.jwt.exception;
+
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.auth0.jwt.exceptions.TokenExpiredException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+@Slf4j
+public class JwtExceptionHandler {
+
+    @ExceptionHandler(JWTVerificationException.class)
+        public ResponseEntity<String> handleJWTVerificationException(JWTVerificationException e) {
```

Comment:
please annotate this method with 
```java
@ResponseStatus(HttpStatus.UNAUTHORIZED)
```
and return String or some object from it directly,
no need to wrap it up into a `ResponseEntity`.